eusurdf/euslisp/eusscene_to_world.l

```diff
@@ -0,0 +1,121 @@
+  (format s "<?xml version=\"1.0\" ?>~%")
+  (format s "<!-- This is a source file of world. -->~%")
+  (format s "<!-- Do not edit this file. -->~%")
+  (format s "<sdf version=\"1.4\">~%")
+  (format s "  <world name=\"default\">~%")
+  (format s "    <physics type=\"ode\">~%")
+  (format s "      <gravity>0 0 -9.81</gravity>~%")
+  (format s "      <ode>~%")
+  (format s "        <solver>~%")
+  (format s "          <type>quick</type>~%")
+  (format s "          <iters>50</iters>~%")
+  (format s "          <sor>1.4</sor>~%")
+  (format s "        </solver>~%")
+  (format s "        <constraints>~%")
+  (format s "          <cfm>0.0</cfm>~%")
+  (format s "          <erp>0.2</erp>~%")
+  (format s "          <contact_max_correcting_vel>100.0</contact_max_correcting_vel>~%")
+  (format s "          <contact_surface_layer>0.0</contact_surface_layer>~%")
+  (format s "        </constraints>~%")
+  (format s "      </ode>~%")
+  (format s "      <real_time_update_rate>1000</real_time_update_rate>~%")
+  (format s "      <max_step_size>0.001</max_step_size>~%")
+  (format s "    </physics>~%")
+  (format s "    <!-- A global light source -->~%")
+  (format s "    <include>~%")
+  (format s "      <uri>model://sun</uri>~%")
+  (format s "    </include>~%")
+  (format s "    <!-- A ground plane -->~%")
+  (format s "    <include>~%")
+  (format s "      <uri>model://ground_plane</uri>~%")
+  (format s "    </include>~%")
+  (format s "    <!-- A robot -->~%")
+  (format s "    <!-- Objects -->~%"))
+
+(defun write-world-file-footer (s)
+  (format s "  </world>~%")
+  (format s "</sdf>~%"))
+
+(defun write-empty-world-file (world-file-path)
+  (with-open-file (f world-file-path :direction :output :if-exists :overwrite)
+    (write-world-file-header f)
+    (write-world-file-footer f)))
+
+(defun add-model-include (world-file-path model &key (name-suffix "") (uri-suffix "_static"))
+  (let ((pos-string (string-join " " (coerce (scale 0.001 (send model :worldpos)) cons)))
+        (rpy-string (string-join " " (reverse (coerce (car (rpy-angle (send model :worldrot))) cons)))))
+    (unix:system (format nil "sed -i -e \"s@    <!-- Objects -->@    <!-- Objects -->\\n    <include>\\n      <name>~A~A</name>\\n      <uri>model://~A~A</uri>\\n      <pose>~A ~A</pose>\\n    </include>@g\" ~a"
+                         (send model :name) name-suffix
+                         (send model :name) uri-suffix
+                         pos-string rpy-string
+                         world-file-path))))
+
+(defun write-model-include (s model &key model-name (suffix "_static") (indent 2))
+  (let ((pos-string (string-join " " (coerce (scale 0.001 (send model :worldpos)) cons)))
+        (rpy-string (string-join " " (reverse (coerce (car (rpy-angle (send model :worldrot))) cons))))
+        (indent-string (make-indent-string indent)))
+    (format s "~A<include>~%" indent-string)
+    (when model-name
+      (format s "~A  <name>~A</name>~%" indent-string model-name))
```

Comment:
> But current conversion flow from rbrain to irteus generates the same model both with namespace and w/o namespace. 

Do you mean that if irt scene is generated with current generation method (convert from rbrain with script), conversion to gazebo urdf and world files should work correctly? If so, it will be OK.